packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -865,6 +948,15 @@ export default class GoogleCalendarService implements Calendar {
     }
   }
 
+  /**
+   * Better alternative to `getPrimaryCalendar` that doesn't require any argument
+   */
+  async fetchPrimaryCalendar() {
```

Comment:
Didn't remove getPrimaryCalendar as it was used at some other place but I believe this is the right way to fetch a primary calendar as it uses authedCalendar